### PR TITLE
Add antio

### PR DESCRIPTION
**Pull request by mscheltienne**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mscheltienne** [comment]
@larsoner Would you agree to also be listed as maintainer on this recipe?

Also, I'm seeing the following warnings in the linux build:

```
WARNING: Setting build arch. This is only useful when pretending to be on another arch, such as for rendering necessary dependencies on a non-native arch. I trust that you know what you're doing.
WARNING: Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
```

I wonder if I need something like:

```
requirements:
  build:
    - python                                 # [build_platform != target_platform]
    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Last point, knowing that the `tests` and data files are not packaged,do you know what would be a good approach to test the recipe?

File: recipes/antio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  license_file:
+    - LICENSE
+    - src/libeep/LICENSE
+    - src/libeep/LICENSE.addendum
+  dev_url: https://github.com/mscheltienne/antio
+
+extra:
+  recipe-maintainers:
+    - mscheltienne
+    - larsoner
```
Comment (larsoner) [inline recipes/antio/meta.yaml:60]: I agree to be a maintainer

**Comment by larsoner** [comment]
> @larsoner Would you agree to also be listed as maintainer on this recipe?

Yep!

> Last point, knowing that the tests and data files are not packaged in the sdist, do you know what would be a good approach to test the recipe?

You can do for example

https://github.com/conda-forge/mne-qt-browser-feedstock/blob/520312adb50af1487d0f1833a92084a429a1697c/recipe/meta.yaml#L43-L45

But it would be great if you could call some method that ensures that the compiled module can actually be accessed. Maybe at some point add to the C code a little function that just returns a string of the libeep library version or something if it doesn't exist already?

**Comment by larsoner** [comment]
> I wonder if I need something like: [cross-python]

Yes you almost certainly do, e.g.

https://github.com/conda-forge/openmeeg-feedstock/blob/6d33a431476df68c8d0dbd49e1ca41c3a8989466/recipe/meta.yaml#L22-L23

**Comment by mscheltienne** [comment]
@conda-forge/help-python-c Ready for review!

**Comment by mscheltienne** [comment]
I was not expecting to break it.. turns out `#include <pybuffer.h>` is missing in those build environment.. https://github.com/python/cpython/blob/main/Include/pybuffer.h

**Comment by mscheltienne** [comment]
@larsoner Any idea? I don't get that one as the source for the `python` recipe includes this header..

**Review by xhochy — APPROVED** [review]

**Pull request merged**